### ISSUE TITLE
Glitch in addon manager compact view

### ISSUE THREAD

**Issue by bege10** [Aris-t2/CustomCSSforFx#418]
1. Issue description:
If an addon has no description the buttons are not in a correct position
This code fixes the issue:
```
/* korrigiert Lage der Buttons von Aris "addonlists_compact.css" bei fehlernder Add-on-Beschreibung */
.addon-description {
     min-height: 20px !important;
}
```

2. Steps to reproduce the issue:

I use these CSS files:
addonlists_badges_old_size.css
addonlists_colorized_button_icons.css
addonlists_compact.css
addonlists_show_addon_version_number.css
addonlists_show_buttons_instead_of_menu_popup_compact_view.css

3. SCREENSHOT(S) of the issue (drag & drop images into this post):

![Screenshot_280](https://user-images.githubusercontent.com/1184237/134263016-ebd4a5cb-186e-4465-adf1-f5423dbc625e.png)
Fixed:
![Screenshot_281](https://user-images.githubusercontent.com/1184237/134263027-caf1e946-a305-40bc-8498-02310403c6c5.png)

4. 
- does the issue occur with latest CustomCSSforFx release (yes / no): yes

5. System information
- OS & OS version: Windows 10 Professional 64 bit
- OS theme (if not default): 
- DPI / HiDPI resolution (e.g. 100%, 125%...): 125%
- Firefox version: 92.0
- Firefox (lw) theme (e.g. default, dark, 'theme name/link'): default
- This projects userChrome.css/userContent.css 'settings' the issue occurs with:

**Comment by Aris-t2** [comment]
Thanks.

**Comment by Aris-t2** [comment]
[Fix is mow part of 4.1.0 v2.](https://github.com/Aris-t2/CustomCSSforFx/releases/tag/4.1.0)
